### ISSUE TITLE
I cant find the predictions

### ISSUE THREAD

**Issue by amlarraz** [splashblot/cvjena-darknet#3]
Hi! When i perform a prediction, yolo works perfectly and show me "prediction in: 0.823465 seconds" but, when i go to view the new jpg with the bounding box and everything i cant find it! I think is my knowledge problem because maybe the save-folder configuration is different in this yolo...i am right? Thanks in advance!

**Comment by jjmata** [comment]
No, no ... you're not going crazy, I've seen that happen myself.  I've even debugged it with some `printf()` statements in the past to see what had been detected and gave up.  Can you add the image you are having that issue with here?  You can just drag/drop it to the text area and it attaches.  Thanks!

**Comment by amlarraz** [comment]
Here is!! Thanks!

![pic14](https://cloud.githubusercontent.com/assets/17318832/19828046/394a0828-9dbc-11e6-9a94-821d1e5fa4e5.jpg)
